server/schema.sql

```diff
@@ -0,0 +1,60 @@
+DROP DATABASE IF EXISTS reviewsservice;
+CREATE DATABASE reviewsservice;
+
+\c reviewsservice;
+
+CREATE TABLE
+  reviews(
+    id SERIAL PRIMARY KEY NOT NULL,
+    product_id SERIAL NOT NULL,
+    rating smallint,
+    date bigint NOT NULL,
+    summary TEXT,
+    body TEXT,
+    recommend boolean,
+    reported boolean,
+    reviewer_name VARCHAR(255),
```

Comment:
Was there a reason you went with `VARCHAHR(255)` here instead of just using TEXT?